app/bundles/SmsBundle/EventListener/CampaignSubscriber.php

```diff
@@ -100,6 +100,7 @@ public function onCampaignTriggerAction(CampaignExecutionEvent $event)
         $lead = $event->getLead();
 
         if ($this->leadModel->isContactable($lead, 'sms') !== DoNotContact::IS_CONTACTABLE) {
+
```

Comment:
No - habit because of symfony code standards; but actually the standard say to only use a blank line before return if it's not alone so technically I'm violating this standard :-/ When we run php css parser, it should get cleaned up

`Add a blank line before return statements, unless the return is alone inside a statement-group (like an if statement);`